nemo/collections/common/callbacks/callbacks.py

```diff
@@ -30,3 +30,7 @@ def on_epoch_end(self, trainer, pl_module):
         curr_time = time.time()
         duration = curr_time - self.epoch_start
         trainer.logger.log_metrics({"epoch_time": duration}, step=trainer.global_step)
+
+        if '_last_lr' in dir(trainer.lr_schedulers[0]['scheduler']):
```

Comment:
What is the point of this line?

Reply:
Logging learning rate.
It shoudn't be in the commit though. I will undo it in the next commit.